src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml

```diff
@@ -14,6 +15,15 @@
     var allowResetPassword = (await SiteService.GetSiteSettingsAsync()).As<ResetPasswordSettings>().AllowResetPassword;
     var loginProviders = (await SignInManager.GetExternalAuthenticationSchemesAsync()).ToList();
     var disableLocalLogin = (await SiteService.GetSiteSettingsAsync()).As<LoginSettings>().DisableLocalLogin;
+
+    // When the value of the RememberMe field is set to anything else than "true" or "false" by the client, which is
+    // the case with automated cracking attempts, then otherwise we'd get a FormatException from the model binder.
+    if (!ViewContext.ModelState.IsValid &&
```

Comment:
This is a workaround solely for the `<input asp-for="RememberMe" class="form-check-input" tabindex="3">` line below, that's where the exception is thrown, due to using the auto-generation of the editor with `asp-for`. With a different implementation of the field (like by having a checkbox implemented directly) the exception won't be thrown. So, for other implementations of this view the code wouldn't necessarily be appropriate.

Reply:
What did you think about? Alternatives I tried but didn't like better:

- Nullable `RememberMe` but that would need a hand-crafted checkbox because the tag helper doesn't support nullable booleans.
- Hand-crafted checkbox by itself.
- Custom model binding but that would be just ugly.